CODEOWNERS

```
@@ -1 +1,2 @@
 * @actions/actions-runtime
+/src @color/tide
```

```suggestion
* @color/tide @chrisdchow
```

Let's just default this to have you and TIDE own all of it for now.